spec/test_templates/json/iam_role/iam_role_with_administratoraccess_policy.json

```diff
@@ -0,0 +1,23 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Resources": {
+      "RoleWithAdministratorAccessPolicy": {
+        "Type": "AWS::IAM::Role",
+        "Properties": {
+          "AssumeRolePolicyDocument": {
+            "Version" : "2012-10-17",
+            "Statement": [ {
+              "Effect": "Allow",
+              "Principal": {
+                "Service": [ "ec2.amazonaws.com" ],
+                "AWS" : "arn:aws:iam::324320755747:root"
+              },
+              "Action": ["sts:AssumeRole"]
+            } ]
+          },
+          "ManagedPolicyArns" : [ "arn:aws:iam::aws:policy/AdministratorAccess" ]
```

Comment:
this is likely good enough given how much other testing there is against IAM roles, but generally i would suggest also including a "positive" test whereby your prove the rule/violation _won't_ be triggered.